<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RealTimePlot</class>
 <widget class="QWidget" name="RealTimePlot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="main_layout">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="error_label">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);
margin-bottom: 5px;
font-sie: 24px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">margin-top: 10px;</string>
         </property>
         <property name="text">
          <string>Activ Parameter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="activ_parameter">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">margin-top: 10px;</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">margin-left: 10px;
color: rgb(255, 0, 255);
margin-top: 10px;</string>
         </property>
         <property name="text">
          <string>Passiv Parameter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="passiv_parameter">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">margin-top: 10px;</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">margin-left: 10px;
margin-top: 10px;</string>
         </property>
         <property name="text">
          <string>Actual Rate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="rate">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">margin-top: 10px;</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
